_config.yml

```diff
@@ -0,0 +1,7 @@
+collections:
+  services:
+    output: true
+    permalink: revops/:collection/:name
+  posts:
+    output: true
+    permalink: revops/:collection/:name
```

Comment:
I think it might make sense i blog posts just live at `/:blog/:name` so we can have a single blog for everything. Then we can tag or categorize blog posts to make them appear on particular pages, e.g. /revops could get blog posts filtered by having the revops tag and show a few of them on the page.

Reply:
We (me and Simon) have originally assumed that the blog and posts will be part of revops only, if the is plan to make it cover a broader spectrum of topics then of course it makes sense to redo the navigation and it's logic and make it work on it's own and just categorize revops related posts accordingly